dlk/tests/test_optimizer.py

```diff
@@ -13,888 +13,497 @@
-            Uint32(),
-            packer.run(qdata2),
-            packed=True,
-            actual_shape=[3, 2, 2, 3]
-        )
-
-        q2 = QTZ_binary_mean_scaling(
-            'qtz2',
-            [3, 2, 2, 3],
-            Float32(),
-            {'input': w2}
-        )
-        q2.scaling_factor = scaling2
-
-        conv2 = Conv(
-            'conv2',
-            [1, 3, 3, 3],
-            Float32(),
-            {'X': aq, 'W': w2},
-            kernel_shape=[2, 2],
-            quantized=True,
-        )
-        conv2.quantizer = q2
-
-        scaling3, qdata3 = self.binary_mean_scaling(data3)
-        w3 = Constant(
-            'weight2',
-            Uint32(),
-            packer.run(qdata3),
-            packed=True,
-            actual_shape=[3, 2, 2, 3]
-        )
-
-        q3 = QTZ_binary_mean_scaling(
-            'qtz3',
-            [3, 2, 2, 3],
-            Float32(),
-            {'input': w3}
-        )
-        q3.scaling_factor = scaling3
-
-        conv3 = Conv(
-            'conv3',
-            [1, 3, 3, 3],
-            Float32(),
-            {'X': aq, 'W': w3},
-            kernel_shape=[2, 2],
-            quantized=True
-        )
-        conv3.quantizer = q3
-
-        y1 = Output(
-            'output1',
-            [1, 3, 3, 3],
-            Float32(),
-            {'input': conv2}
-        )
-
-        y2 = Output(
-            'output2',
-            [1, 3, 3, 3],
-            Float32(),
-            {'input': conv3}
-        )
+        # activation quantizer
+        s1 = Constant('aq_const1', Float32(), np.array(1))
+        s2 = Constant('aq_const2', Float32(), np.array(2))
+        aq = QTZ_linear_mid_tread_half('aqtz1', [1, 4, 4, 3], Float32(), {'X': conv1, 'Y': s1, 'Z': s2})
+
+        # Conv2
+        w2 = Constant('weight2', Float32(), data2)
+        kq = QTZ_binary_mean_scaling('kqtz1', [1, 2, 2, 3], Float32(), {'input': w2})
+        conv2 = Conv('conv2', [1, 3, 3, 3], Float32(), {'X': aq, 'W': kq}, kernel_shape=[2, 2])
+        conv2.a_quantizer = [aq]
+        conv2.quantizer = kq
+
+        # One output
+        y = Output('output', [1, 3, 3, 3], Float32(), {'input': conv2})
+
+        # add ops to the graph
+        graph.add_op_and_inputs(y)
+
+        return graph
+
+
+class TestPassQuantizeConvolutions(unittest.TestCase):
+    """Test class for packing weight."""
+    def test_pass_quantize_convolutions(self) -> None:
+        """Test pass."""
+        data1 = np.float32(np.random.rand(1, 2, 2, 3))
+        data2 = np.float32(np.random.rand(1, 2, 2, 3))
+        graph1 = self.create_sample_graph(data1, data2)
+
+        pass_quantize_convolutions(graph1)
+
+        self.assertEqual(graph1.get_op('aqtz1').dtype, QUANTIZED_NOT_PACKED(),
+                         '[Failed] Found output dtype of activation quantizer not proper')
+        self.assertEqual(graph1.get_op('conv2').dtype, Float32(),
+                         '[Failed] Found output dtype of conv not proper')
+
```

Comment:
It is better to add below line.
```
        self.assertEqual(graph1.get_op('kqtz1').dtype, Uint32, ...)
```

Reply:
Included the assertion mentioned above in the test with commit https://github.com/blue-oil/blueoil/pull/53/commits/ca65f4ec73ad368f8aa65d11c1911e82fa593930

Reply:
:+1: